domain/src/main/kotlin/no/nav/su/se/bakover/domain/brev/BrevInnhold.kt

```diff
@@ -14,6 +14,14 @@ import no.nav.su.se.bakover.domain.brev.beregning.BrevTilbakekrevingInfo
 abstract class BrevInnhold {
     fun toJson(): String = objectMapper.writeValueAsString(this)
 
+    /**
+     * Tar med property'en "erAldersbrev" når toJson kalles. Vi kan ikke sette verdien som en property, siden
+     * da er sakstypen alltid UFØRE
+     */
+    fun isErAldersbrev(): Boolean {
```

Comment:
Holder vel med enten `is` eller `er`?

Reply:
Tolker kommentaren din her som at uføre og alder nå er implementert i det samme brevet? Hadde kanskje sett for meg at det var like greit om det vare bare var to helt separate brev med forskjellig template og innhold?

Reply:
> Holder vel med enten `is` eller `er`?

Nei, ikke hvis man vil ha den automatiske mappingen fra objectMapper til property'en `erAldersbrev`


> Tolker kommentaren din her som at uføre og alder nå er implementert i det samme brevet? Hadde kanskje sett for meg at det var like greit om det vare bare var to helt separate brev med forskjellig template og innhold?

Slik som de ser ut nå er brevene veldig like, og måten de bruker partials på gjør at man må i så fall duplisere noen av de for å få riktig brevtekst. I alle fall for vedtak ser det sånn ut, du må gjerne se på PR'en på su-pdfgen for å se hvordan brevene blir lagd

Reply:
Jackson sin automatiske mapping 😢 Kan vi bruke @JsonProperty eller noe for å sikre oss et ønsket navn både i domenet og i json her? Oooog ha en test som sjekker denne sjuke mappinga 😆

Reply:
Haha, ser på å få det gjort på en annen måte 😅